### PR TITLE
Add loading page

### LINKED ISSUE #74: Add loading page
Add a page which should be shown while the page is being loaded.

### PR DESCRIPTION
**Pull request by illseption**
- Added loading page using css and jQuery
- Added jQuery to base.html to run loader animation
- Loading page opens before every page of main website 
Fixes #74

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -46,7 +68,6 @@
   </div>
 
   <p align="justify"><font size="2px" color="grey"><center>Copyright 2017 &copy; Programming Club,UIET. All Rights Reserved. <br></center></font></p>    
-</footer>
+</footer>
```
Comment (divyam3897) [inline templates/base.html]: Remove this change.

File: templates/base.html [hunk 2]
```diff
@@ -46,7 +68,6 @@
   </div>
 
   <p align="justify"><font size="2px" color="grey"><center>Copyright 2017 &copy; Programming Club,UIET. All Rights Reserved. <br></center></font></p>    
-</footer>
+</footer> 
   </body>
-
```
Comment (divyam3897) [inline templates/base.html]: Remove this change.

File: templates/base.html [hunk 3]
```diff
@@ -17,12 +17,34 @@
     <link rel = 'stylesheet' href="{% static 'home/base.css' %}">
     <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet"/>
     <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css" rel="stylesheet"/>
+
+    <!-- jQuery -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```
Comment (divyam3897) [inline templates/base.html]: add the js to static/ and include it here

Comment (illseption) [reply templates/base.html]: to static/admin/js or staticFiles/home?

**Pull request closed without merging**